src/flint/fmpz_mat.jl

```diff
@@ -885,13 +885,16 @@ mutable struct lll_ctx
    end
 end
 
-
 @doc raw"""
     lll_with_transform(x::ZZMatrix, ctx::lll_ctx = lll_ctx(0.99, 0.51))
 
-Compute a tuple $(L, T)$ where $L$ is the LLL reduction of $a$ and $T$ is a
-transformation matrix so that $L = Ta$. All the default parameters can be
-overridden by supplying an optional context object.
+Compute a tuple $(L, T)$ where $L$ is the LLL reduction of the
+$\mathbb{Z}$-basis given by the columns of the matrix $x$ and $T$ is a
```

Comment:
Is "columns" correct here? I assumed so because the transformation matrix is multiplied from the left, but I could not find any concrete evidence in the flint documentation.

Reply:
I think multiplying from left means row operation?

Reply:
But if I have just one vector in the lattice and multiply the vector from the left by the transformation matrix, then the vector must be given by a column? I'm confused.